BridgeAppSDK/SBAConsentReviewStep.swift

```diff
@@ -251,5 +255,29 @@ class SBAConsentReviewStepViewController: ORKPageStepViewController {
         
         return stepResult
     }
+    
+    var consentReviewResult: SBAConsentReviewResult? {
+        return self.result?.results?.findObject({ $0 is SBAConsentReviewResult}) as? SBAConsentReviewResult
+    }
+    
+    // Override the default method for goForward and attempt user registration. Do not allow subclasses
+    // to override this method
+    final override func goForward() {
+        
+        // Check that the user has consented or fail with an error
+        guard let consentResult = self.consentReviewResult where consentResult.isConsented else {
+            let error = NSError(domain: "SBAConsentReviewStepDomain",
+                                code: -1,
+                                userInfo: [NSLocalizedDescriptionKey : Localization.localizedString("SBA_REGISTRATION_NOT_CONSENTED")])
+            self.delegate?.stepViewControllerDidFail(self, withError: error)
+            return
+        }
+        
+        // set the consent to the shared user
+        sharedUser.consentSignature = consentResult.consentSignature
```

Comment:
Some studies (e.g. the resilience project) are going to want to have the user sign multiple separate consent forms. Not saying we have to support this today, just food for thought.